runtime/JavaScript/src/antlr4/Lexer.d.ts

```diff
@@ -16,13 +16,16 @@ export declare class Lexer extends Recognizer<number> {
     _tokenStartLine: number;
     _tokenStartColumn: number;
     _type: number;
+    _modeStack: number[];
+    _mode: number;
 
     constructor(input: CharStream);
     reset(): void;
     nextToken(): Token;
     skip(): void;
     more(): void;
     more(m: number): void;
+    mode(m: number): void;
```

Comment:
I guess the intent with the 2nd `more` was to be `mode`. Can you merge these ?

Reply:
The 2 `more` are already in the current [Lexer.d.ts](https://github.com/antlr/antlr4/blob/3c06d0ac6f8750c252ec496eb3629b0a6e9e47fb/runtime/JavaScript/src/antlr4/Lexer.d.ts#L24C5-L25C27).

Reply:
There should only be 1, it's a bug in the existing file.